lang/nl.js

```diff
@@ -2,6 +2,7 @@
 
 export default {
 	"components.alert.close": "Waarschuwing sluiten",
+	"components.breadcrumbs.breadcrumb": "Breadcrumb",
```

Comment:
Oh weird I guess we don't have a Dutch translation?

Reply:
Yeah it seems like it. I can do it myself though . not sure if that's a good idea though

Reply:
Nah it's OK -- once this merges the Serg.io automatic translation stuff should kick in and it'll get translated. 👍